VTEX - Orders API.json

```diff
@@ -15944,3158 +15499,1581 @@
-                    "example": "7"
-                  },
-                  "productRefId": {
-                    "type": "string",
-                    "nullable": true,
-                    "description": "Product Ref ID of the item replacing the previous one.",
-                    "example": "smallcup"
-                  },
-                  "refId": {
-                    "type": "string",
-                    "nullable": true,
-                    "description": "Reference ID of the item replacing the previous one.",
-                    "example": "356"
-                  },
-                  "ean": {
-                    "type": "string",
-                    "nullable": true,
-                    "description": "European Article Number (EAN) of the item replacing the previous one.",
-                    "example": "8745121641680"
-                  },
-                  "name": {
-                    "type": "string",
-                    "nullable": true,
-                    "description": "Name of the Product associated with the item replacing the previous one.",
-                    "example": "Small cup of coffee with pink logo"
-                  },
-                  "skuName": {
-                    "type": "string",
-                    "nullable": true,
-                    "description": "Name of the SKU replacing the previous one.",
-                    "example": "Small blue cup"
-                  },
-                  "modalType": {
-                    "type": "string",
-                    "nullable": true,
-                    "description": "A [modal](https://help.vtex.com/en/tutorial/how-does-the-modal-work--tutorials_125) attaches an unusual product, such as meat or glass, to a carrier specialized in shipping it.",
-                    "example": "GLASS"
-                  },
-                  "parentItemIndex": {
-                    "type": "integer",
-                    "nullable": true,
-                    "description": "Parent item index of the item replacing the previous one.",
-                    "example": 0
-                  },
-                  "parentAssemblyBinding": {
+                    "checkedInPickupPointId": "storeNameExample_901",
+                    "cancellationData": {
+                        "RequestedByUser": true,
+                        "RequestedBySystem": false,
+                        "RequestedBySellerNotification": false,
+                        "RequestedByPaymentNotification": false,
+                        "Reason": "Item was too big in the client.",
+                        "CancellationDate": "2022-10--05T15:40:33"
+                    },
+                    "clientPreferencesData": {
+                        "locale": "en-US",
+                        "optinNewsLetter": false
+                    }
+                }
+            },
+            "Createchange": {
+                "required": [
```

Comment:
adicionar `description`

Reply:
The request body first level does not require a description.